### PR TITLE
Fix description for method ActiveRecord::ConnectionAdapters::SchemaStatements#add_timestamps [ci skip]

### PR DESCRIPTION
**Pull request by bogdanvlviv**

### CODE REVIEW

File: activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb [hunk 1]
```diff
@@ -1051,9 +1051,9 @@ def columns_for_distinct(columns, orders) # :nodoc:
       end
 
       # Adds timestamps (+created_at+ and +updated_at+) columns to +table_name+.
-      # Additional options (like <tt>null: false</tt>) are forwarded to #add_column.
+      # Additional options (like <tt>null: true</tt>) are forwarded to #add_column.
```
Comment (robin850) [inline activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb]: Actually I think both are true ; this method forward all the given options, it just sets `:null` to `false` if the option isn't yet specified. What do you think about writing `Additional options (like +:null+) are forwarded` ?

Comment (bogdanvlviv) [inline activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb]: You right, it will be better.

**Comment by prathamesh-sonpatki** [comment]
@bogdanvlviv Can you squash the commits?

**Comment by bogdanvlviv** [comment]
Yes, sure.

**Comment by bogdanvlviv** [comment]
@prathamesh-sonpatki, i did it.

**Comment by vipulnsward** [comment]
Thanks @bogdanvlviv !

**Pull request merged**